iroh-bitswap/src/lib.rs

```diff
@@ -77,21 +77,16 @@ pub struct Bitswap<S: Store> {
     _workers: Arc<Vec<JoinHandle<()>>>,
 }
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Default)]
 enum PeerState {
     Connected(ConnectionId),
     Responsive(ConnectionId, ProtocolId),
     Unresponsive,
+    #[default]
```

Comment:
Just so I understand: this is not really related to the PR, right?

Keep it in, it seems reasonable...

Reply:
Correct, clippy complained about it so I fixed it. Not sure what changed with clippy to start complaining about it but figured it was a good fix anyway.